packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1215,6 +1215,7 @@ class CupertinoActionSheetAction extends StatefulWidget {
     required this.onPressed,
     this.isDefaultAction = false,
     this.isDestructiveAction = false,
+    this.cursor = MouseCursor.defer,
```

Comment:
Wondering if we can use `mouseCursor` instead of `cursor` to be consistent with other material/cupertino APIs that offer a configurable mouse cursor. For example, refer to uses of [mouseCursor](https://github.com/search?q=repo%3Aflutter%2Fflutter%20this.mouseCursor&type=code) vs. that of [cursor](https://github.com/search?q=repo%3Aflutter%2Fflutter+this.cursor&type=code).

Reply:
sure I can update it to use mouseCursor instead of cursor for consistency

Reply:
making as resolved taken care in most recent PR